### DISCUSSION TITLE
help in Installation

### DISCUSSION THREAD

**Discussion by Shubham00-ai** [Eleven-Trading/TradeNote#59] [Q&A]
can anybody help me to install tradenote 
i dont know how its done 
i dont know that much programming 
can you please simplify the steps and help me

**Comment by fabricio-fn** [comment]
I'm a programmer and I don't know what this "tradenote" is, can you be more specific?

**Comment by gitricko** [comment]
Try https://github.com/gitricko/tradenote-selfhost
